scripts/build/companion_libs/050-zlib.sh

```diff
@@ -94,20 +94,20 @@ do_zlib_backend() {
         "${extra_config[@]}"
 
     CT_DoLog EXTRA "Building zlib"
-    CT_DoExecLog ALL make ${JOBSFLAGS}
+    CT_DoExecLog ALL ${make} ${JOBSFLAGS}
```

Comment:
Seems the make change isn't needed; I'll remove it from the next set of changes for this PR.